mungegithub/submit-queue/nginx-redirect/nginx.conf

```diff
@@ -5,6 +5,23 @@ events {
 http {
   server {
     listen 80;
+    server_name community.submit-queue.k8s.io;
+    location / {
+        if ($http_user_agent ~ (GoogleHC) ) {
+          return 200;
+        }
+        if ($http_x_forwarded_proto != 'https') {
+          return 301 https://$host$request_uri;
+        }
+        proxy_pass http://community-sq-status;
+        proxy_set_header Host $host;
```

Comment:
can you add another commit to hoist these proxy_set_header directives up to the http scope? should reduce duplication in this file